src/dmd/mtype.d

```diff
@@ -238,7 +238,7 @@ private enum TFlags
     char_        = 0x40,
 }
 
-enum ENUMTY : int
+enum ENUMTY : ubyte
```

Comment:
Does `ENUMTY` still need to exist on the D side? Can't we replace `ENUMTY` with `TY` everywhere in `*.d`?

Reply:
Yes, we can.

Reply:
I know if we accept ENUMTY, then all the other enums will be refactored to be ENUMXX. So, no.

Reply:
Err, what? ENUMTY already exists, that was a suggestion to get rid of `ENUMTY`...

For reference `ENUMTY` is [used only in objc_glue, and impcnvtab](https://github.com/dlang/dmd/search?q=ENUMTY&unscoped_q=ENUMTY). I'm all for getting rid of it.